src/picknik_ur_mock_hw_config/description/picknik_ur_machine_tending.xacro

```diff
@@ -56,7 +51,7 @@
     script_filename="$(find ur_robot_driver)/resources/ros_control.urscript"
     output_recipe_filename="$(find ur_robot_driver)/resources/rtde_output_recipe.txt"
     input_recipe_filename="$(find ur_robot_driver)/resources/rtde_input_recipe.txt">
-    <origin xyz="0 0 0.8" rpy="0 0 ${pi/2}" />
+    <origin xyz="0 0 0" rpy="0 0 ${pi/2}" />
```

Comment:
Lol. I think the grid is fine, it makes some sense to have the robot be the center of the universe, anyway.